packages/amplify_authenticator/lib/src/mixins/authenticator_radio_field.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:amplify_authenticator/src/widgets/form_field.dart';
+
+import 'package:flutter/material.dart';
+
+mixin AuthenticatorRadioField<FieldType, FieldValue,
+        T extends AuthenticatorFormField<FieldType, FieldValue, T>>
+    on AuthenticatorFormFieldState<FieldType, FieldValue, T> {
+  void initState() {
+    selectionValue = widget.inputconfig!.selections[0].value as FieldValue;
```

Comment:
I personally think it makes sense to preselect, since it could potentially save the end user an action and also obviates the need for a validator. But I'm fine with moving to null as an initial value if we agree that's better.